lib/edit/p_info.txt

```diff
@@ -398,8 +396,8 @@ C:a:k:+0:+400:Magic
 C:a:k:+0:+500:Nature
 C:a:k:+0:+500:Divination
 C:a:k:+0:+500:Conveyance
-C:a:k:+0:+700:Disarming
-C:a:k:+0:+50:Sneakiness
+
```

Comment:
I wonder if removing these opens up new options for Rogue-types. Probably not since they were never 100% reliable... and so nobody would actually max out Disarming/Sneakiness and just Magic-Device + Staff/Rod.